llms/mlx_lm/models/stablelm.py

```diff
@@ -198,20 +197,12 @@ def __call__(
         self,
         x: mx.array,
         mask: mx.array = None,
-        cache: mx.array = None,
-    ) -> Tuple[mx.array, mx.array]:
+        cache=None,
```

Comment:
I don't personally care but in most cases this is written as `cache: Optional[Any] = None`

Reply:
Good catch, will fix.